app/components/collections/show/header_component.html.erb

```diff
@@ -0,0 +1,17 @@
+<div class="d-flex justify-content-between">
```

Comment:
Instead of sharing the header_component across works and collections, split it out to more easily accommodate collection-specific buttons and behavior. Also allows testing with collections and works separately.